cmd/kubeadm/app/phases/selfhosting/selfhosting_test.go

```diff
@@ -134,7 +134,7 @@ spec:
         - --service-cluster-ip-range=10.96.0.0/12
         - --tls-cert-file=/etc/kubernetes/pki/apiserver.crt
         - --kubelet-client-certificate=/etc/kubernetes/pki/apiserver-kubelet-client.crt
-        - --advertise-address=$(HOST_IP)
+        - ""
```

Comment:
If we're doing this, can we do it so that this line is removed instead of replacing it with just `""`?